Core/Core/Courses/CourseListView.swift

```diff
@@ -56,15 +67,45 @@ public struct CourseListView: View {
     }
 
     var empty: some View {
-        EmptyViewRepresentable(
+        EmptyView.AsView(
             title: NSLocalizedString("No Courses", bundle: .core, comment: ""),
             body: NSLocalizedString("It looks like there aren’t any courses associated with this account. Visit the web to create a course today.", bundle: .core, comment: ""),
             imageName: "PandaTeacher"
-        )
+        ).testID("empty")
+    }
+
+    @ViewBuilder
+    func list<Content: View>(@ViewBuilder content: () -> Content) -> some View {
+        if useList {
```

Comment:
was this just for testing things? I don't love needing to do this.

Reply:
This is because List and properties work very badly together. I filed a bug with Apple, but I'm not sure what else we can do about it.